testkit/src/main/java/akka/kafka/testkit/internal/AlpakkaKafkaContainer.java

```diff
@@ -172,7 +172,9 @@ public String getJmxServiceUrl() {
   @Override
   protected void doStart() {
     withCommand(
-        "sh", "-c", "while [ ! -f " + STARTER_SCRIPT + " ]; do sleep 0.1; done; " + STARTER_SCRIPT);
+        "sh",
+        "-c",
+        "while [ ! -f " + START_STOP_SCRIPT + " ]; do sleep 0.1; done; " + START_STOP_SCRIPT);
```

Comment:
So this other script is included already, right?

Reply:
Yes, it just wasn't referenced in this start command.